numpy/core/src/umath/_umath_tests.c.src

```diff
@@ -564,7 +564,7 @@ UMath_Tests_test_signature(PyObject *NPY_UNUSED(dummy), PyObject *args)
         core_dim_sizes = Py_None;
     }
     Py_DECREF(f);
-    return Py_BuildValue("iOOOO", core_enabled, core_num_dims,
+    return Py_BuildValue("iNNNN", core_enabled, core_num_dims,
```

Comment:
Yikes, how can our tests have passed?

Reply:
It only leaks a few reference counts, pretty harmless in a way ;).